_data/opencollective.json

```diff
@@ -43,34 +43,34 @@
   {
     "name": "Will Lewis",
     "last_payment": 5.0,
-    "all_time": 510.0,
+    "all_time": 520.0,
     "since": "Jul 12, 2022"
   },
   {
-    "name": "SportOnlineBetting",
+    "name": "https://sportonlinebetting.net/review/slotsnbets-casino/",
     "last_payment": 5.0,
-    "all_time": 15.0,
+    "all_time": 20.0,
     "since": "Jul 4, 2022"
   },
   {
     "name": "Stayatcasinos.com",
     "url": "https://stayatcasinos.com/",
     "last_payment": 5.0,
-    "all_time": 15.0,
+    "all_time": 20.0,
     "since": "Jul 4, 2022"
   },
   {
     "name": "Bromine0x23",
     "url": "https://twitter.com/bromine0x23",
     "last_payment": 10.0,
-    "all_time": 240.0,
+    "all_time": 260.0,
```

Comment:
I've noticed this happening sometimes that `all_time` amount increases by two times `last_payment`. Payments should be monthly and the update rate is more frequent than that.
So why is it double the amout? 🤔 
@beta-ziliani Any idea?

Reply:
I see that OC is sending this one user duplicated (with different user ids but same name). But in the list of transactions there is only a sum of $130, so it seems like an issue on OC. Will ping them.

Reply:
It's not just this user, I've definitely seen it with others as well.